pyproject.toml

```diff
@@ -0,0 +1,76 @@
+[project]
+name = "formula1-race-analysis"
+version = "0.0.2"
+description = "Add your description here"
+readme = "README.md"
+authors = [{name = "KrisKulbych", email = "k.kulbych@gmail.com"}]
+requires-python = ">=3.12"
+dependencies = ["formula1_race_analysis",]
+
+[dependency-groups]
+dev = [
+    "pre-commit>=4.0.1",
+    "pytest-cov>=6.0.0",
+    "pytest>=8.3.3",
+    "ruff>=0.8.1",
+    "mypy>=1.13.0",
+    "click>=8.1.7",
+    "structlog>=24.4.0",
+]
+
+[tool.uv.workspace]
+exclude = ["src/tests/*"]
+
+[tool.pytest.ini_options]
+addopts = "-v --cov=. --cov-fail-under=90"
+testpaths = [
+    "tests",
+]
+
+[tool.coverage.run]
+omit = [
+    "tests/*",
+]
+
+[tool.ruff]
+line-length = 120
+
+[tool.ruff.lint]
+select = ["ALL"]
+ignore = ["F401", "D", "S", "TRY", "T", "EM", "DTZ", "G"]
+
+[tool.ruff.format]
+indent-style = "space"
+
+[tool.mypy]
+python_version = "3.12"
+ignore_missing_imports = true
+disallow_untyped_calls = true
+disallow_untyped_defs = true
+disallow_incomplete_defs = true
+check_untyped_defs = true
+warn_no_return = true
+show_error_context = true
+show_column_numbers = true
+warn_unused_configs = true
+
+[tool.flake8]
+ignore = ["COM812", "ISC001"]
+
+[build-system]
+requires = ["setuptools>=61", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+include-package-data = true
+
+[tool.setuptools.packages.find]
+where = ["src"]
+
+[tool.setuptools.exclude-package-data]
+"tests" = ["*"]
+
+[[tool.uv.index]]
+name = "testpypi"
+url = "https://test.pypi.org/simple/"
```

Comment:
![image](https://github.com/user-attachments/assets/131e3eb4-e247-489e-9f0d-3b3af6729197)
I can't sync packages because not all of them available in test index, we need also extra index url

Reply:
Still an issue
```
[[tool.uv.index]]
name = "index"
url = "https://pypi.org/simple"
publish-url = "https://test.pypi.org/legacy/"
```
Technically you don't need testpypi for packages locally since you are not dependant on any package from test pypi. But publish should happen to test pypi